manifest.php

```diff
@@ -40,7 +40,7 @@
     'label' => 'Item core extension',
     'description' => 'TAO Items extension',
     'license' => 'GPL-2.0',
-    'version' => '10.19.0',
+    'version' => '10.19.1',
```

Comment:
do you think we really need this or did you forget to commit ? 
because i see no changes

Reply:
Already pushed npm dependencies update